core/src/subgraph/context.rs

```diff
@@ -177,10 +174,13 @@ impl<C: Blockchain, T: RuntimeHostBuilder<C>> IndexingContext<C, T> {
         self.instance.causality_region_next_value()
     }
 
-    #[cfg(debug_assertions)]
     pub fn instance(&self) -> &SubgraphInstance<C, T> {
         &self.instance
     }
+
+    pub fn hosts(&self) -> &[Arc<T::Host>] {
```

Comment:
There's already an instance fn, why not just make the field public?

Reply:
done